deploy.sh

```diff
@@ -68,15 +68,20 @@ if [ $BRANCH != "master" ] ; then
     node_modules/.bin/emu-algify --throwing-indicators < $BRANCH_DIR/index.intermediate.html > $BRANCH_DIR/index.html
     rm $BRANCH_DIR/index.intermediate.html
     echo "Branch snapshot output to $WEB_ROOT/$BRANCHES_DIR/$BRANCH"
-else
+fi
+#else
     # Living standard, if master
     curl https://api.csswg.org/bikeshed/ -f -F file=@index.bs \
          -F md-Text-Macro="SNAPSHOT-LINK $SNAPSHOT_LINK" \
          > $WEB_ROOT/index.intermediate.html
     node_modules/.bin/emu-algify --throwing-indicators < $WEB_ROOT/index.intermediate.html > $WEB_ROOT/index.html
     rm $WEB_ROOT/index.intermediate.html
+
+    cp service-worker.js $WEB_ROOT/service-worker.js
+    cp manifest.json $WEB_ROOT/manifest.json
```

Comment:
Hmm. I can't see the benefit of using a different file extension. In the end I might drop this commit though, as the manifest gives almost no benefit it seems.